backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/i18n/po/POTrlRepository.java

```diff
@@ -89,26 +96,51 @@ public static String toTrlTableName(final String tableName)
 		return tableName + TRL_TABLE_SUFFIX;
 	}
 
-	public final POTrlInfo createPOTrlInfo(final String tableName, @Nullable final String keyColumnName, final List<String> translatedColumnNames)
+	public static String toBaseTableName(final String tableName)
 	{
-		if (keyColumnName == null || Check.isEmpty(keyColumnName))
+		if(!tableName.endsWith(TRL_TABLE_SUFFIX))
 		{
-			return POTrlInfo.NOT_TRANSLATED;
+			return "";
 		}
+		final String baseTableName = StringUtils.replace(tableName, TRL_TABLE_SUFFIX, "");
+		return tableDAO.isExistingTable(baseTableName) ? baseTableName : "";
+	}
 
-		if (translatedColumnNames.isEmpty())
+	public final POTrlInfo createPOTrlInfo(final String tableName, final ImmutableList<String> keyColumnNames, final List<String> translatedColumnNames)
```

Comment:
can u give me an example of a table with composed primary key and which have *_Trl table support?

Reply:
the _trl table itself

Reply:
the purpose of POTrlInfo is to describe a base table, tell which is the trl table, what columns are translated etc,

and not to have a POTrlInfo for each table, no matter if is a base table or a trl table